### ISSUE TITLE
Modules to be converted--Entire list

### ISSUE THREAD

**Issue by dcwhite** [SCIInstitute/SCIRun#1746]
TODO

**Comment by jansonap** [comment]
I'd like to add "WriteBundle" to this list under "List: Bundles" and request that full bundle support/conversion be moved up the list in priority of implementation (if that ranking exists)

**Comment by dcwhite** [comment]
Will do. At some point, I noted that WriteBundle was mistakenly missing from this list. Thanks for the reminder.

**Comment by dcwhite** [comment]
@jessdtate Intern work?

**Comment by jessdtate** [comment]
Yeah, this is something they can tackle for sure.  Perhaps when QT5 is running smoother.

**Comment by dcwhite** [comment]
**Analysis (2026-06-12)**

Master checklist tracking module conversion to .module JSON config files (related to issue #101). Active work is on branch `module-config-cleanup`. 

Per current memory: 84 modules remained as of batch 4; next up is BuildSurfaceLaplacianMatrix. This issue is the tracking vehicle for that work — no separate action needed here.

**Comment by dcwhite** [comment]
**Data point for this list:** a full v4.7→v5 parity scan is now in #2615, weighted by usage across the 434 shipped v4 example networks.

Short version: raw coverage is 33%, but **155 missing and 87 dormant modules appear in zero shipped v4 networks**. The usage-weighted gap is **69 modules** — 38 missing, 31 dormant — which is a very different planning target than the raw counts.

Two pieces broken out: #2614 (v4 volume-rendering pipeline, dormant, the largest coherent gap) and #2613 (parameter-level parity audit — the current comparison is name-level only, so a module ported with fewer options still counts as "present").
